src/libslic3r/GCode.cpp

```diff
@@ -4515,6 +4515,29 @@ static std::unique_ptr<EdgeGrid::Grid> calculate_layer_edge_grid(const Layer& la
     return out;
 }
 
+struct SlopedLoop
```

Comment:
Can this class merged with ExtrusionPathSloped(or potentially ExtrusionSlopedLoop if we made it to derive from ExtrusionLoop) class for simplicity?

Reply:
@SoftFever to whom were the questions directed?

Reply:
The dev ;)